File: MPI/mpicheck/tests/functions/2.0/MPI_Info_set.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Info var_0;
    const char *var_1;
    const char *var_2;
    int ret;
    /* calls */
    ret = MPI_Info_set(var_0, var_1, var_2);
    ret = PMPI_Info_set(var_0, var_1, var_2);
    return 0;
}

File: MPI/mpicheck/tests/functions/2.0/MPI_Info_set.f90

        
        program main
        use mpi
        
        INTEGER var_0
       CHARACTER*(10) var_1
       CHARACTER*(10) var_2
       INTEGER var_3
        call mpi_info_set(var_0, var_1, var_2, var_3)
       call pmpi_info_set(var_0, var_1, var_2, var_3)
        end program main
    